1657 documentation needs to make clear that the soil model is a topsoil model (#1688)
ImperialCollegeLondon/virtual_rainforest: docs/source/virtual_ecosystem/theory/soil/environmental_links.md

fit into any of the cases mentioned already, and so it will be discussed in a separate
section.

## Finding soil and litter layer temperatures

Before we discuss the different classes of environmental response, we need to mention

> how the environment itself is determined. The temperatures and soil waster potentials

Reviewer: ```suggestion
how the environment itself is determined. The temperatures and soil water potentials
```

are calculated using the abiotic and hydrology models which use their own layer
definitions that do not correspond to the depth of the biotically active topsoil.

:::{note}

> The Virtual Ecosystem uses two different definitions of "topsoil", the uppermost layer

Reviewer 2: Can you refer to `docs/source/_static/images/layer_structure.svg` here?

of the hydrology model and the entire simulated depth of the soil model. This allows the
depth of soil that is considered to be (microbially active) topsoil to be altered by
users, without forcing them to update the entire setup of the hydrology model to match.

> In places in the documentation where both definitions of topsoil are pertinent the soil

Reviewer 2: ```suggestion
In places in the documentation where both definitions of topsoil are pertinent, the soil
```

> model specific definition of topsoil is referred to as the "biotic topsoil".

Reviewer: I find this a bit confusing. Would there be a way around the 'topsoil', maybe 'biotic soil zone'? That would keep it clearly distinct from the model layer approach

Reviewer 2: Instead of zone, maybe just consistently "layer".

Author: "layer" has a specific meaning in the model layer structure (which is used by abiotic, plants and hydrology), litter and soil sit on top of that structure but don't use the exact layers. So I can't really use the word layer as it introduces confusion

Author: > I find this a bit confusing. Would there be a way around the 'topsoil', maybe 'biotic soil zone'? That would keep it clearly distinct from the model layer approach

Completely agree that I need a different name, issue I have with "biotic soil zone" is kind of the same as I had with "maximum depth of microbial activity", in that it sounds like we are making a strong claim about where there is microbial activity in the soil, rather than choosing to only simulate a subset of the total activity (because we can't simulate it all and the upper soil makes the biggest contribution)

I was thinking of going with something like "microbial simulation depth" instead, as that explains the situation more clearly (we are making a decision not to simulate below a certain depth)

:::

For the above-ground litter, things are pretty straightforward as the air temperature of
the layer immediately above the soil surface is just used. (As this litter is
above-ground, soil water potential is not relevant).

The below-ground litter is assumed to be spread out evenly over the topsoil, so the
process for calculating the relevant soil temperatures is identical to that of the

> (biotic) topsoil (which we discuss below).

Reviewer 2: I find "(biotic) topsoil" still a bit confusing, is it the BS layer in the layer figure?

The (biotic) topsoil is simulated as a single stratum (of user configurable) depth. This

> stratum uses a homogeneous environment, e.g. a single temperature is used for its entire

Reviewer: here you could also use 'zone' ?

Reviewer 2: maybe just "layer"

Author: going to use "zone" as it isn't already in use for another purpose ("layer" is used for `abiotic/hydrology/plants`, "stratum" is used by `animal`)

Reviewer 2: Actually zone could work since we do say "root zone".

extent. The environmental variables for this stratum are calculated as follows:

$$
T = \sum^N_{i=1} f_i *\frac{V_{i-1} + V_{i}}{2}
$$

> where $N$ is the number of soil layers, $f_i$ is the fraction of soil layer $i$ that

Reviewer 2: The number of soil layers, is it 2, or 3?

lies within the (biotic) topsoil, and $V_{i}$ is the value of the environmental variable

> of interest at depth $i$. No values are generated for "layer zero", so for each variable

Reviewer 2: I don't quite get the "layer zero" sentence here.

> we need to choose a value for it. For temperature, we chose $T_0$ to be the air

Reviewer 2: ```suggestion
we need to choose a value for it. For temperature, we chose layer-zero temperature $T_0$ to be the air
```

temperature of the layer immediately above the soil surface. For soil water potential,

> we set $\psi_0 = \psi_1$, which amounts to assuming that the soil water potential does

Reviewer 2: ```suggestion
we set layer-zero soil water potential to be the same as that of the first layer, $\psi_0 = \psi_1$, to assume that the soil water potential does
```

calculated using the abiotic and hydrology models which use their own layer definitions
that do not exactly correspond to the simulation depth of the soil-microbial model.

:::{note}

In the Virtual Ecosystem "topsoil" currently refers to the uppermost layer of the
hydrology model (see the [layer structure
diagram](../../implementation/core_components_overview.md#the-vertical-layer-structure)
for more details). The depth that the soil-microbial model is simulated to can be
altered independently of this topsoil depth, meaning that it can be changed without
having to update the entire setup of the hydrology model to match.

:::

For the above-ground litter, things are pretty straightforward as the air temperature of
the layer immediately above the soil surface is just used. (As this litter is
above-ground, soil water potential is not relevant).

The below-ground litter is assumed to be spread out evenly over the simulated depth of
the soil-microbial model, so the process for calculating the relevant soil temperatures
is identical to that of the soil microbes (which we discuss below).

The zone the soil-microbial model is simulated for is of user configurable depth and is

> assumed to be homogeneous. This means that for each environmental variable and average

Reviewer 2: ```suggestion
assumed to be homogeneous. This means that for each environmental variable an average
```
